### PR TITLE
derive: set implicits globally

### PR DESCRIPTION
**Pull request by Blaisorblade**
Same bugfix as in #450, now extended to all remaining uses; this other use was introduced in #328 with the same bug.

~~Completely untested.~~ Now checks for this problem in all tests from #328.

### CODE REVIEW

**Comment by gares** [comment]
Great!
Would you mind adding a line to the Changes file?

**Comment by gares** [comment]
(I've fixed nix CI in master, if you could rebase that would give a green light as well)

File: Changelog.md [hunk 1]
```diff
@@ -282,8 +286,8 @@ Requires Elpi 1.14.1 and Coq 8.15.
 Requires Elpi 1.13.6 and Coq 8.15.
 
 ### APPS
-- `derive Inductive i {A}` now correctly sets `A` implicit status
-- `lock Definition f {A}` now correctly sets `A` implicit status
+- `derive Inductive i {A}` now correctly sets `A` implicit status (XXX)
+- `lock Definition f {A}` now correctly sets `A` implicit status (XXX)
```
Comment (Blaisorblade) [inline Changelog.md:290]: I can just drop this, but should we add a note here since we found later problems?

**Comment by Blaisorblade** [comment]
I don't get the DOC failure — spurious? But the rest works!

**Review by gares — APPROVED** [review]

File: Changelog.md (hunk 1 above)
Comment (Blaisorblade) [reply Changelog.md]: In doubt, dropped again.

**Pull request merged**